DllImportGenerator/DllImportGenerator/StubCodeGenerator.cs

```diff
@@ -70,24 +117,15 @@ public override (string managed, string native) GetIdentifiers(TypePositionInfo
             }
         }
 
-        public static (BlockSyntax Code, MethodDeclarationSyntax DllImport) GenerateSyntax(
-            IMethodSymbol stubMethod,
-            IEnumerable<TypePositionInfo> paramsTypeInfo,
-            TypePositionInfo retTypeInfo)
+        public (BlockSyntax Code, MethodDeclarationSyntax DllImport) GenerateSyntax()
         {
-            Debug.Assert(retTypeInfo.IsNativeReturnPosition);
-            
-            var context = new StubCodeGenerator(Stage.Setup);
-
+            this.CurrentStage = Stages[0];
```

Comment:
This is the second location we set the initial value. We then set it again in the loop below. It is a bit confusing when this is important to be set and when it isn't.